src/ProgramBuilder.ts

```diff
@@ -33,10 +33,10 @@ export class ProgramBuilder {
      */
     public allowConsoleClearing = true;
 
-    public options: BsConfig;
+    public options: FinalizedBsConfig = util.normalizeConfig({});
     private isRunning = false;
-    private watcher: Watcher;
-    public program: Program;
+    private watcher: Watcher | undefined;
+    public program: Program | undefined;
```

Comment:
This may also be a breaking change, but I don't see any good way around that

Reply:
I think this is fine. It's technically true.